modules/DataLoadManager.py

```diff
@@ -157,3 +149,28 @@ def start_single_import(self, target_engine, model_file, requested_full_refresh)
         data_load_tracker.completed_successfully()
         self.data_load_tracker_repository.save(data_load_tracker)
         self.logger.info(f"Import Complete for: {model_name}. {data_load_tracker.get_statistics()}")
+
+    @staticmethod
```

Comment:
Is there a reason `*` is a parameter?

Reply:
Yes, some python magic. @seanbudd the magician.

Reply:
ensures that you need to supply everything as keyword arguments (you *have* to call it like chintan did)